cpp/open3d/pipelines/integration/UniformTSDFVolume.h

```diff
@@ -52,6 +52,8 @@ class UniformTSDFVolume : public TSDFVolume {
                    const Eigen::Matrix4d &extrinsic) override;
     std::shared_ptr<geometry::PointCloud> ExtractPointCloud() override;
     std::shared_ptr<geometry::TriangleMesh> ExtractTriangleMesh() override;
+    /// Temporary function will be deleted/"merged" with ExtractTriangleMesh
+    std::shared_ptr<geometry::TriangleMesh> ExtractTriangleMesh_v2();
```

Comment:
It would be better to inject the implementation as an option in `ExtractTriangleMesh(bool use_vtk = False)`.